apps/dashboard/src/components/workflow-editor/in-app-preview.tsx

```diff
@@ -174,6 +175,10 @@ const Markdown = (props: MarkdownProps) => {
 
   const tokens = useMemo(() => parseMarkdownIntoTokens(children || ''), [children]);
 
+  if (children === PreviewIssueEnum.PREVIEW_ISSUE_REQUIRED_CONTROL_VALUE_IS_MISSING) {
+    return null;
+  }
```

Comment:
this part ensures that the issue returned by the API from the backend is not rendered.  
I have two questions:  
1. Is this the correct approach to render null it in the new dashboard?  
2. Why are we returning this value from the backend in the first place? Don’t we already use issues to inform the frontend when there’s a problem?